docs/source/collect/logs.md

```diff
@@ -38,6 +38,22 @@ The duration of the maximum oldest log to include.
 ##### `limits.maxLines`
 The number of lines to include, starting from the newest.
 
+## Flags
```

Comment:
Hmm... this feels like the wrong place to document this. These are command line flags, not spec flags. Maybe these should be documented in a new page about the CLI tools, and then a comment added here which links to that new page.

Reply:
A 'CLI tools' page sounds useful, I could add the other flags `interactive`, `format`, `collector-pullpolicy`, etc... Any other troubleshoot cli tool that could be added?

Reply:
To be clear, I'm not suggesting a page to talk about other tools, but how to run `kubectl preflight` and `kubectl support-bundle` and what CLI options these tools support. Maybe it's two pages, one for each command.

Reply:
It doesn't really fit cleanly in the current table of contents, unfortunately.